package.json

```diff
@@ -20,7 +20,10 @@
     "babel": "^5.8.23",
     "babel-core": "^5.8.25",
     "babel-loader": "^5.3.2",
+    "bcrypt": "^0.8.5",
```

Comment:
`bcrypt-nodejs` is a pure JavaScript implementation of the bcrypt library, and is the one I usually use, but it might not really matter for our purposes